avocado/core/nrunner.py

```diff
@@ -359,6 +359,31 @@ class BaseRunner(abc.ABC):
     def __init__(self, runnable):
         self.runnable = runnable
 
+        # set default Avocado env variables
+        #
+        # NOTE: these environment variables are redefined inside an Avocado
+        # instrumented test.
+        #
+        # this is a code duplication from avocado/core/version.py to avoid
+        # nrunner depending on other than python modules
+        version = "unknown.unknown"
+        if PKG_RESOURCES_AVAILABLE:
+            try:
+                version = pkg_resources.get_distribution(
+                    'avocado-framework').version
+            except pkg_resources.DistributionNotFound:
+                pass
+        os.environ['AVOCADO_VERSION'] = version
```

Comment:
I won't mind if you split this code in a own method `def _get_avocado_version()`.

Reply:
Ack!